recipes/biopet-vcfstats/1.1/meta.yaml

```diff
@@ -0,0 +1,45 @@
+# Based on OpenJDK recipe in conda-forge
+# https://github.com/conda-forge/openjdk-feedstock/blob/master/recipe/meta.yaml
+# This file was automatically generated by the sbt-bioconda plugin.
+
+     
+test:
+  commands:
+  - biopet-vcfstats --version
+  - biopet-vcfstats --help
+package:
+  name: biopet-vcfstats
+  version: '1.1'
+source:
+  url: https://github.com/biopet/vcfstats/releases/download/v1.1/VcfStats-assembly-1.1.jar
+  sha256: f9ad47d5d840a99271d5f70b30046304330f1a1bf6e99756da5ecd2841da4a32
+build:
+  number: 0
+requirements:
+  run:
+  - openjdk
+  - python
+  build: []
+extra:
+  notes: 'biopet-vcfstats is a Java program that comes with a custom wrapper shell
```

Comment:
You might want to add a note that this recipe is autogenerated and should not be updated manually?

Reply:
Do you have a DOI that you can attach to the extras section?

Reply:
> You might want to add a note that this recipe is autogenerated and should not be updated manually?
Great idea. Already a comment  on topin the header says that it has been automatically generated but I will add a big WARNING as well.
> Do you have a DOI that you can attach to the extras section?
Not for this specific tool, but I will add it as a key to the plugin, so it will be easy to add in the future. Thanks for the great suggestion!